models/voodoo_trade/base/voodoo_trade_base_schema.yml

```diff
@@ -0,0 +1,89 @@
+version: 1
```

Comment:
I think we may keep all .yml version as `2`:
https://docs.getdbt.com/reference/project-configs/version#yml-property-file-versions

Reply:
Hello @Hosuke,
I have updated version `1` to version `2` in all .yml files. Please review and let me know if anything else is required from my end.